src/main/java/com/company/TAM2/Main.java

```diff
@@ -0,0 +1,47 @@
+package com.company.TAM2;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    public static final int COLUMNS_COUNT = 2;
+
+    public static void main(String[] args) {
+        String fromFileName =  "orange.csv";
+        String toFileName = "result.txt";
+        getStatistic(fromFileName, toFileName);
+    }
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        try (
```

Comment:
please refactor the body of this method getStatistic().
divide the solution by a three smaller methods.
- readFromFile() that reads data from file
- createReport() that processing the readed data and creates the result.
- writeToFile() that writes the result report to the file.

You can use appropriate collections (like List) to pass the data between methods and  optimizing the solution.

Reply:
fixed